File: library/src/main/java/com/pengrad/telegrambot/model/request/InlineQueryResultCachedSticker.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * Stas Parshin
 * 06 May 2016
 */
public class InlineQueryResultCachedSticker extends InlineQueryResult<InlineQueryResultCachedSticker> implements Serializable {
    private final static long serialVersionUID = 0L;

    private String sticker_file_id;

    public InlineQueryResultCachedSticker(String id, String fileId) {
        super("sticker", id);
        this.sticker_file_id = fileId;
    }
}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/InlineQueryResultCachedSticker.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 06 May 2016
 */
class InlineQueryResultCachedSticker(id: String, private val sticker_file_id: String) : InlineQueryResult<InlineQueryResultCachedSticker>("sticker", id), Serializable {
    companion object {
        private const val serialVersionUID = 0L
    }
}
